ci/cudf_pandas_scripts/run_tests.sh

```diff
@@ -53,3 +65,22 @@ python -m pytest -p cudf.pandas \
     --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cudf-pandas-coverage.xml" \
     --cov-report=term \
     ./python/cudf/cudf_pandas_tests/
+
+output=$(python ci/cudf_pandas_scripts/fetch_pandas_versions.py $pandas_version_constraint)
+
+# Remove the brackets and spaces from the output to get a comma-separated list
+output=$(echo $output | tr -d "[] \'\'")
```

Comment:
Just make the Python script output what you want!

Reply:
Done